### ISSUE TITLE
SingleLinePerSelector warns with wrong line number

### ISSUE THREAD

**Issue by hail2u** [brigade/scss-lint#565]
This reproduced when ruleset had multiline selector like this:

```css
a,
b,
c,
d,
e
f,
g {
  display: auto;
}
```

```bash
$ scss-lint test.scss
test.scss:2 [W] SingleLinePerSelector: Each selector in a comma sequence should be on its own single line
```

scss-lint detects error correctly, but should warn line number `5` (or `6`?).

Tested with Ruby 2.2.2 (x64) and scss_lint v0.41.

**Comment by srawlins** [comment]
This one could be tough to fix. I think we're getting bad data from Sass. Using the test case above, and printing out every node in `visit_sequence`, I see that _each_ node is supposedly on line 1!

```
$ bundle exec bin/scss-lint issue-565.scss
Sass::Selector::Sequence [a] on line: 1
Sass::Selector::Sequence ["\n", b] on line: 1
Sass::Selector::Sequence ["\n", c] on line: 1
Sass::Selector::Sequence ["\n", d] on line: 1
Sass::Selector::Sequence ["\n", e, "\n", f] on line: 1
Sass::Selector::Sequence ["\n", g] on line: 1
issue-565.scss:2 [W] SingleLinePerSelector: Each selector in a comma sequence should be on its own single line
```

:frowning:

**Comment by sds** [comment]
Yeah, we'll need to count the `\n` characters and increment the line number appropriately (or submit a fix upstream). My experience tells me it might be easier to fix it on our side.

**Comment by sds** [comment]
Fixed in 596b96d4672a4efad4503fe02c4f0289cc5860f0.
